src/sentry/static/sentry/app/views/settings/components/settingsSearch/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import keydown from 'react-keydown';
+import styled from 'react-emotion';
+
+import {t} from '../../../../locale';
+import InlineSvg from '../../../../components/inlineSvg';
+import Search from '../../../../components/search';
+
+const MIN_SEARCH_LENGTH = 1;
+const MAX_RESULTS = 10;
+
+class SettingsSearch extends React.Component {
+  @keydown('/')
+  handleFocusSearch(e) {
+    if (!this.searchInput) return;
+    if (e.target === this.searchInput) return;
+
+    e.preventDefault();
+    this.searchInput.focus();
+  }
+
+  render() {
+    return (
+      <Search
+        {...this.props}
+        minSearch={MIN_SEARCH_LENGTH}
+        maxResults={MAX_RESULTS}
+        renderInput={({getInputProps}) => (
+          <SearchInputWrapper>
+            <SearchInputIcon size="14px" />
+            <SearchInput
+              innerRef={ref => (this.searchInput = ref)}
+              {...getInputProps({
+                type: 'text',
+                placeholder: t('Press "/" to search settings'),
```

Comment:
not sure about this placeholder, especially when it's already focused, but its good to advertise this somewhere